### ISSUE TITLE
Vertica ingestion does not work

### ISSUE THREAD

**Issue by igor-kramer** [open-metadata/OpenMetadata#2678]
**Affected module**
Ingestion Framework

**Describe the bug**
I'm trying to ingest tables/views from Vertica and receive errors like this

```
[2022-02-08 13:40:28,954] ERROR    {metadata.ingestion.sink.metadata_rest:211} - Failed to ingest table TickStore in database ikramer
[2022-02-08 13:40:28,954] ERROR    {metadata.ingestion.sink.metadata_rest:217} - Cannot deserialize value of type `java.util.UUID` from Object value (token `JsonToken.START_OBJECT`)
 at [Source: (org.glassfish.jersey.message.internal.ReaderInterceptorExecutor$UnCloseableInputStream); line: 1, column: 1441] (through reference chain: org.openmetadata.catalog.api.data.CreateTable["database"])
[2022-02-08 13:40:28,954] INFO     {metadata.ingestion.source.sql_source_common:33} - Table Scanned: vertica-test.TickStore
[2022-02-08 13:40:36,855] ERROR    {metadata.ingestion.sink.metadata_rest:211} - Failed to ingest table ml_prophets_prediction in database ikramer
[2022-02-08 13:40:36,855] ERROR    {metadata.ingestion.sink.metadata_rest:217} - Cannot deserialize value of type `java.util.UUID` from Object value (token `JsonToken.START_OBJECT`)
 at [Source: (org.glassfish.jersey.message.internal.ReaderInterceptorExecutor$UnCloseableInputStream); line: 1, column: 2843] (through reference chain: org.openmetadata.catalog.api.data.CreateTable["database"])
```

Table is simple:

```
-- auto-generated definition
create table TickStore
(
    ts       timestamp,
    symbol   varchar(8),
    bid      float default 1.00,
    test_col int   default 20,
    column_5 varchar(20)
);

comment on table TickStore is 'My Tick table';
```

**To Reproduce**

Run ingesting process on Vertica.

**Expected behavior**
Vertica table are created in OpenMetadata UI

**Version:**
 - Python version:
 - OpenMetadata version: 0e73de4658ac6cacf7f4cda90434ec5d34b01d07
 - OpenMetadata Ingestion package version: 0e73de4658ac6cacf7f4cda90434ec5d34b01d07

**Additional context**

**Comment by igor-kramer** [comment]
@ulixius9 how can i check it?
```
mvn clean install
make install
make install_dev
```

**Comment by ulixius9** [comment]
@igor-kramer right, after install_dev 
```
make generate
cd ingestion
pip install . 
pip install '.[vertica]' 
metadata ingest -c examples/workflows/vertica.json
```

**Comment by igor-kramer** [comment]
@ulixius9 error is still here
I'll try clean open-metadata (0.8) installation tommorow and will inform you

**Comment by harshach** [comment]
@ulixius9 can we publish a dev package, @ayush-shah

**Comment by harshach** [comment]
@igor-kramer please make sure you are running the open-metadata server from main instead of 0.8 if you are using the metadata framework from main.
@ulixius9 probably easier to cherry-pick this change into 0.8 branch and publishing a dev subsequently publishing it to main pypi after the confirmation from @igor-kramer

**Comment by igor-kramer** [comment]
@ulixius9 in 0.8.0 i have got error
![image](https://user-images.githubusercontent.com/2264016/153179127-cd2aa455-f80d-40e6-a3b1-5d153ddc78f1.png)

In UI i see strange datatypes  (date(1), timestamp(1), numeric(1), int(1))
![image](https://user-images.githubusercontent.com/2264016/153180451-47817a9f-b098-4645-b6ce-ee2a5ed33bbd.png)


Table DDL:
```
create table ml_prophets_prediction
(
    user_uid                   uuid not null,
    fact_date                  date not null,
    toxic_prediction_prob      numeric(37, 15),
    profitable_prediction_prob numeric(37, 15),
    updated_at                 timestamp,
    deleted_at                 timestamp,
    etl_created_at             timestamp,
    etl_updated_at             timestamp,
    etl_crc                    int,
    partner_uid                varchar(64),
    constraint ml_prophets_prediction_pk
        primary key (fact_date, user_uid)
);
```

**Comment by igor-kramer** [comment]
Fresh 0.9-dev installation with the same tables

![image](https://user-images.githubusercontent.com/2264016/153192984-8900429c-4a15-4c43-9008-39a1099e97a9.png)

**Comment by ayush-shah** [comment]
@ulixius9 can you take this issue up?

**Comment by ulixius9** [comment]
Sure @ayush-shah

**Comment by ulixius9** [comment]
I will fix the incorrect datatypes (date(1), timestamp(1), numeric(1), int(1)) also add the UUID data type in json schema 
and will also fix the import errors in [vertica.py](https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/ingestion/source/vertica.py)

**Comment by harshach** [comment]
@igor-kramer just an fyi 
https://github.com/open-metadata/OpenMetadata/pull/2709#pullrequestreview-878999329

we map source (Vertica) types into standard types on OM side https://github.com/open-metadata/OpenMetadata/blob/main/catalog-rest-service/src/main/resources/json/schema/entity/data/table.json#L36 if we don't find a type we default to varchar.
In UUID case we didn't list that as a type on OM side. We are adding this in the main. 
So you can use this against the 0.8 but UUID will show up as archer
@ulixius9 once the PR is merged lets cherry-pick into 0.8 as well.

**Comment by igor-kramer** [comment]
@harshach thanks!  We don't use OpenMetadata in production right now (we are testing different apps before beginning datacatalog implementation) and can wait 0.9
